random-ballerina/Package.md

```diff
@@ -0,0 +1,3 @@
+## Module Overview
+
+This module provides functions to related to random number generation.
```

Comment:
Shall we describe more about the operations we support in this module

Reply:
Updated in f6d19bc